decidim-dev/lib/tasks/locale_checker.rake

```diff
@@ -18,4 +18,14 @@ namespace :decidim do
       end
     end
   end
+
+  desc "Allows a decidim installation to sanitize its available locales"
+  task sanitize_locales: :environment do
+    Decidim::Organization.find_each do |organization|
+      sanitized_locales =
+        organization.available_locales & Decidim.available_locales.map(&:to_s)
```

Comment:
I'm not sure about this. Shouldn't we update processes, etc in order to fix the locales there too?